pkg/blockdeviceclaim/v1alpha1/blockdeviceclaim.go

```diff
@@ -114,7 +117,7 @@ func (bdc *BlockDeviceClaim) AddFinalizer(finalizer string) (*ndm.BlockDeviceCla
 
 	bdc.Object.Finalizers = append(bdc.Object.Finalizers, finalizer)
 
-	bdcAPIObj, err := NewKubeClient().
+	bdcAPIObj, err := NewKubeClient(WithKubeConfigPath(bdc.configPath)).
```

Comment:
_n, why is the `WithKubeConfigPath` not in the next line as a method call?

Reply:
being lesser than 80 chars, placed it in the same line..
I know there are other lines greater than 80 chars, but, done to some extent

Reply:
80 chars is one. also it let us know we are using the builder pattern.